GNUmakefile

```diff
@@ -68,6 +100,31 @@ pretest: lint vet fmtcheck
 test: pretest
 	$(GO) test -cover -v ./... || exit;
 
+test-fetch: test-fetch-cleanup start-fetch-postgres start-fetch-mysql start-fetch-redis build test-fetch-redis test-fetch-sqlite test-fetch-mysql test-fetch-postgres test-fetch-cleanup
+
+test-fetch-cleanup: kill-fetch-postgres kill-fetch-mysql kill-fetch-redis
+
+start-fetch-postgres:
+	@docker run -d  --name fetch-postgres-oval --restart=always -p 5432:5432 -e "POSTGRES_DB=goval" -e "POSTGRES_PASSWORD=world" -e "POSTGRES_USER=hello" postgres:14.1-alpine
+
+start-fetch-mysql:
+	@docker run -d  --name fetch-mysql-oval --restart=always  -p 3306:3306 -e "MYSQL_DATABASE=goval" -e "MYSQL_PASSWORD=world" -e "MYSQL_USER=hello" -e "MYSQL_RANDOM_ROOT_PASSWORD=true" mysql:8.0
+
+start-fetch-redis:
+	@docker run -d --name fetch-redis-oval -p 6379:6379 redis
+
+kill-fetch-postgres:
+	@docker rm -f fetch-postgres-oval 
+
+kill-fetch-mysql:
+	@docker rm -f fetch-mysql-oval
+
+kill-fetch-redis:
+	@docker rm -f fetch-redis-oval
```

Comment:
If you just want to verify the fetch, I didn't think --restart was necessary.
If you want to kill it, you can simply invoke it with --rm.
The name "start-fetch-*" may mislead people into thinking that just executing the task will start fetch.